linkedList/removingDuplicatesFromSorted.js

```diff
@@ -0,0 +1,18 @@
+var deleteDuplicates = function (head) {
+    if (head == null) {
```

Comment:
linkedList/removeAllnodesOfVal.js linkedList/removingDuplicatesFromSorted.js aur yeh similar hai
ek function kyu nhi pass kar sakta asking which node to delete